### PR TITLE
REM-1139, REM-1140: Fix method get_batch_status with invalid parameter id

### PR DESCRIPTION
**Pull request by anastasiia-bilova**
### Jira references
- First ticket (task) — https://remmeio.atlassian.net/browse/REM-1139
- Second ticket (task) — https://remmeio.atlassian.net/browse/REM-1140

### CODE REVIEW

File: remme/shared/forms/_validators.py [hunk 1]
```diff
@@ -21,3 +21,36 @@ def __call__(self, form, field):
 
             field.errors[:] = []
             raise validators.StopValidation(message)
+
+
+class DataRequired(object):
+    """
+    Checks the field's data is 'truthy' otherwise stops the validation chain.
+    This validator checks that the ``data`` attribute on the field is a 'true'
```
Comment (dmytrostriletskyi) [inline remme/shared/forms/_validators.py]: Blank line after summary line should be presented.

File: remme/shared/forms/_validators.py [hunk 2]
```diff
@@ -21,3 +21,36 @@ def __call__(self, form, field):
...
+    value (effectively, it does ``if field.data``.) Furthermore, if the data
+    is a string type, a string containing only whitespace characters is
+    considered false.
+    If the data is empty, also removes prior errors (such as processing errors)
+    from the field.
+    Args:
```
Comment (dmytrostriletskyi) [inline remme/shared/forms/_validators.py]: The blank line between `Arguments` section, please.

File: remme/shared/forms/_validators.py [hunk 3]
```diff
@@ -21,3 +21,36 @@ def __call__(self, form, field):
...
+
+    def __init__(self, message=None):
+        self.message = message
+
+    def __call__(self, form, field):
+
+        if field.data == 0:
+            pass
+
+        else:
```
Comment (dmytrostriletskyi) [inline remme/shared/forms/_validators.py]: Try https://github.com/Remmeauth/remme-core/pull/255#discussion_r265195302.

File: remme/shared/forms/_validators.py [hunk 4]
```diff
@@ -21,3 +21,36 @@ def __call__(self, form, field):
...
+    message (str): error message to raise in case of a validation error.
+    """
+    field_flags = ('required',)
+
+    def __init__(self, message=None):
+        self.message = message
+
+    def __call__(self, form, field):
+
+        if field.data == 0:
```
Comment (dmytrostriletskyi) [inline remme/shared/forms/_validators.py]: Leave a comment to the documentation why you pass when data is zero.

Comment (anastasiia-bilova) [reply remme/shared/forms/_validators.py]: @dmytrostriletskyi, done.

**Pull request merged**